src/test/java/integration/FileDownloadTest.java

```diff
@@ -2,42 +2,60 @@
 
 import com.codeborne.selenide.Configuration;
 import com.codeborne.selenide.FileDownloadMode;
+import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.openqa.selenium.remote.DesiredCapabilities;
 
 import java.io.File;
 import java.io.IOException;
+import org.openqa.selenium.net.NetworkUtils;
 
+import static com.codeborne.selenide.FileDownloadMode.FOLDER;
+import static com.codeborne.selenide.FileDownloadMode.HTTPGET;
+import static com.codeborne.selenide.FileDownloadMode.PROXY;
 import static com.codeborne.selenide.Selectors.byText;
 import static com.codeborne.selenide.Selenide.$;
-import static com.codeborne.selenide.Selenide.closeWebDriver;
 import static com.codeborne.selenide.Selenide.open;
 import static com.codeborne.selenide.files.FileFilters.withExtension;
 import static java.nio.charset.StandardCharsets.UTF_8;
 import static org.apache.commons.io.FileUtils.readFileToString;
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class FileDownloadTest {
-  @BeforeEach
-  void setUp() {
-    closeWebDriver();
-
-    DesiredCapabilities capabilities = new DesiredCapabilities();
-    capabilities.setBrowserName("chrome");
-    Configuration.browserCapabilities = capabilities;
-    Configuration.fileDownload = FileDownloadMode.FOLDER;
+public class FileDownloadTest extends ITest {
+  @AfterEach
+  void tearDown() {
```

Comment:
Maybe move this method to `ITest`?
And also add `@BeforeEach` annotation to it?